draft-ietf-intarea-provisioning-domains.md

```diff
@@ -558,18 +560,29 @@ is not defined in this document.
 
 When the H-flag of the PvD Option is set, hosts MAY attempt to
 retrieve the PvD Additional Information associated with a given PvD by
-performing an HTTP over TLS {{?RFC2818}} GET query to
+performing an HTTP over TLS {{!RFC2818}} GET query to
 https://\<PvD-ID\>/.well-known/pvd.
 Inversely, hosts MUST NOT do so whenever the H-flag is not set.
 
+Recommendations for how to use TLS securely can be found in {{!RFC7525}}.
+
+When a host retrieves the PvD Additional Information, it MUST
+verify that the TLS server certificate is valid for the performed
+request; specifically, that a DNS-ID {{?RFC6125}} on the certificate is equal to
+the PvD ID expressed as an FQDN. This validation indicates that the
+owner of the FQDN authorizes its use with the prefix advertised by the router.
+If this validation fails, hosts MUST close the connection and treat the PvD
+as if it has no Additional Information.
+
 HTTP requests and responses for PvD additional information use the
 "application/pvd+json" media type (see {{iana}}). Clients
-SHOULD include this media type as an Accept header in their GET
+SHOULD include this media type as an Accept header field in their GET
 requests, and servers MUST mark this media type as their Content-Type
-header in responses.
+header field in responses.
 
-Note that the DNS name resolution of the PvD ID, the PKI (Public Key Infrastructure) checks as
-well as the actual query MUST be performed using the considered PvD.
+Note that the DNS name resolution of the PvD ID, any connections made
+for certficate validation (such as OCSP {{?RFC6960}}), and
+the HTTP request itself MUST be performed using the considered PvD.
```

Comment:
Should we refer to previous text about DNS operations ?

Reply:
We could, although adding the reference here may be a bit awkward for the text. This is also a bit distinct, in that it is referring specifically to what is used for fetching the additional info, which that earlier section doesn't discuss.